config/defaults.js

```diff
@@ -30,6 +30,7 @@ const defaultConfig = {
 		// Disabled plugins
 		shortcuts: {
 			enabled: false,
+			overrideMediaKeys: false,
```

Comment:
It would be better to default it to true for backwards-compatibility
```suggestion
			overrideMediaKeys: true,
```